creator-node/src/snapbackSM/peerSetManager.js

```diff
@@ -153,13 +155,16 @@ class PeerSetManager {
     logger.info(`getNodeUsers() nodeUsers.length: ${nodeUsers.length}`)
 
     // Ensure every object in response array contains all required fields
-    nodeUsers.forEach((nodeUser) => {
+    for (const nodeUser of nodeUsers) {
```

Comment:
was there a reason for this iteration change?

Reply:
yeah it's faster to do basic functions instead of forEach, reduce, etc. this particular optimization is minor, but currently a bottleneck is that there are so many users being processed here